x-pack/test_serverless/api_integration/services/transform/security_common.ts

```diff
@@ -33,14 +37,18 @@ export function TransformSecurityCommonProvider({ getService }: FtrProviderConte
     {
       name: 'transform_dest',
       elasticsearch: {
-        indices: [{ names: ['.slo-*'], privileges: ['read', 'index', 'manage', 'delete'] }],
+        indices: [
+          { names: ['.slo-observability.*'], privileges: TOTAL_INDEX_PRIVILEGE_SET_EDITOR },
```

Comment:
I wanted to align this with the actual index pattern and privileges we specify for the SLO product, but I'm not sure if that's even the purpose of this test/service because it seems like it's intended to just test transforms and doesn't actually really relate to SLOs at all?

Reply:
I believe best person to answer is @mgiota , she added the service

Reply:
@mgiota If this is SLO related then I think this change is fine. The tests are indeed passing.